Single source of truth for timesheeting and updated guidance re 5 hour rule (#662)
madetech/handbook: guides/process/scheduling/how_to_timesheet.md

Where these commitments total less than 5 hours per week, they should be included in our standard operating overhead and should not be accounted for separately from billed hours.

> Note that some internal activities do not count towards this 5 hours as our clients are perceived to benefit from them indirectly. These include:

Reviewer: ```suggestion
Some internal activities do not count towards the 5 hours as our clients benefit from them indirectly. These include:
```

* Hiring
* Line management (1:1s etc.)

> * Taking part in communities of practice

Author: Any more we should add here?

Reviewer 2: I think this is a bit confusing. Hiring *is* included for the purposes of the 5 hour break point, as explicitly stated in the first line of this section. Line management and CoP involvement (including CARE) are not.

Author: Ah, I obviously got the wrong end of the stick from our meeting. kind of proves the value of this exercise! Will remove hiring.

> When the commitment exceeds 5 hours in a week, each element should be allocated individually to the appropriate project in Kimble and not included in billable hours.

Author: It is not clear here whether we start allocating elements above 5 hours, or the whole lot.

If we do 7 hours, to we allocate 7 hours to non client categories or 2 hours?

Reviewer: ```suggestion
When the total internal commitment exceeds 5 hours in a week, each commitment should be allocated in its entirety to the appropriate project in Kimble and not included in billable hours.
```

Author: How about this?
```suggestion
When the commitment exceeds 5 hours in a week the 5 hour rule no longer applies, so each element should be allocated individually to the appropriate project in Kimble and not included in billable hours.
```

Author: Mashup of mine and @adrianclay's...
```suggestion
When the total internal commitment exceeds 5 hours in a week the 5 hour rule no longer applies, so each commitment should be allocated in its entirety to the appropriate project in Kimble and not included in billable hours.
```

Author: I'm going with my mashup.
